packages/azure-http-specs/specs/client/namespace/client.tsp

```diff
@@ -0,0 +1,36 @@
+import "./main.tsp";
+import "@azure-tools/typespec-client-generator-core";
+
+using TypeSpec.Http;
+using Spector;
+using Azure.ClientGenerator.Core;
+using Client.ClientNamespace;
+
+@route("/client/client-namespace")
+namespace ClientNameSpaceClient;
+
+@client({
+  name: "ClientNamespaceFirstClient",
+  service: Client.ClientNamespace,
+})
+@clientNamespace("client.clientnamespace", "java")
+@clientNamespace("client.clientnamespace", "python")
```

Comment:
@v-jiaodi Could you help add js part?

Reply:
pr link: https://github.com/Azure/typespec-azure/pull/2224